src/python/grpcio_tests/tests/qps/qps_worker.py

```diff
@@ -43,11 +43,13 @@ def run_worker_server(driver_port, server_port):
                         type=int,
                         dest='driver_port',
                         help='The port the worker should listen on')
```

Comment:
I feel the help fields for driver_port and server_port aren't making it clear what's the purpose of each of them.
Can we adjust the to clarify?

Reply:
I think this is better. Let me know.